### ISSUE TITLE
Add OMIM 614742 to ORPHA:2032 Idiopathic pulmonary fibrosis

### ISSUE THREAD

**Issue by Orphanet-ES** [OD4RD/Main-Help-Desk#174]
Dear all,

A Spanish coder has suggested we link the OMIM [614742](https://omim.org/entry/614742) "Pulmonary fibrosis and/or bone marrow failure syndrome, telomere-related, 1" to [ORPHA:2032](https://www.orpha.net/consor/cgi-bin/Disease_Search.php?lng=EN&data_id=7029) Idiopathic pulmonary fibrosis, since "_some cases of familial fibrosis could be explained/caused by a TERT variant_".

In fact, the [TERT](https://www.orpha.net/consor/cgi-bin/Disease_Genes.php?lng=EN&data_id=16789) gene is already linked to the disease in Orphanet.

Kind regards, 

María Elena

**Comment by JTorphanet** [comment]
Dear @Orphanet-ES, 

Thank you for your message and indeed you raised a problem/several problems. 
This entity concerns an idiopathic form and it's not logic that we have so many genes and OMIM numbers linked. We have the intuition that this entity should be divided into 2 different. 
Moreover, we will start in the next coming weeks an ERN revision for interstitial lung diseases and this entity will be included. 

In other words, we need to review before adding genes/OMIM to be sure. 

Best regards, 

Julie

**Comment by Orphanet-ES** [comment]
Dear @Mickael-Orphanet, 

Have you any news regarding this issue?

Kind regards, 

María Elena

**Comment by Mickael-Orphanet** [comment]
Dear @Orphanet-ES , 

This is a misleading problem. Idiopathic is from unknown cause but it seems that there is a missusage in the litterature. I will add the OMIM as BTNT to the Idiopathic pulmonary fibrosis. But this needs a nomenclature modification. In any case your issue is now over. 

Have a great day,
Mickael

**Comment by Orphanet-ES** [comment]
Dear @Mickael-Orphanet ,

I understand your reasoning, but as long as you do not create a separate entity for idiopathic disease and for the variant with known causes, it is the only option.

Thanks for your help.

Kind regards, 

María Elena
